packages/core/EventManager.ts

```diff
@@ -566,9 +581,7 @@ export default class EventManager {
        *  fallback to the first connected calendar - Shouldn't be a CRM calendar
        */
       // Backwards compatibility until CRM manager is created
-      const [credential] = this.calendarCredentials.filter(
-        (cred) => !cred.type.endsWith("other_calendar") || !cred.type.endsWith("crm")
```

Comment:
Though this code is being modified now but I think it should have been `!cred.type.endsWith("other_calendar") && !cred.type.endsWith("crm")`